lib/private_address_check.rb

```diff
@@ -6,21 +6,37 @@
 module PrivateAddressCheck
   module_function
 
+module PrivateAddressCheck
+  module_function
+
   CIDR_LIST = [
-    # Loopback
```

Comment:
Please keep comments and segment each addition into specific sections under a comment that explains what it is.

Reply:
@reedloden I plan on adding comments in just a bit. On top of that, this PR doesn't even include tests so it isn't ready for merging quite yet.